src/Agent.Listener/CommandSettings.cs

```diff
@@ -23,6 +23,7 @@ public sealed class CommandSettings
         public bool NoStart => TestFlag(Constants.Agent.CommandLine.Flags.NoStart);
         public bool Unattended => TestFlag(Constants.Agent.CommandLine.Flags.Unattended);
         public bool Version => TestFlag(Constants.Agent.CommandLine.Flags.Version);
+        public bool DeploymentAgent => TestFlag(Constants.Agent.CommandLine.Flags.DeploymentAgent);
```

Comment:
I talked with Bryan, we think this Yes/No question might cause confusion. Since the current agent is used for Build and Release, and Release and Deployment sounds like similar things.
Change the config like this:
QA1: Agent usage type (Build/Releases or Deployment): default to "Build/Releases".  you can change the question a little bit, i am not good and writing. but the idea is not just asking a Yes/No question, provide a full list of option, so user can have idea about what will happen.
QA2: If user choose Deployment agent, only ask project url, otherwise ask serverurl like today. you can do the validation and figure out projectname, collectionname, serverurl by yourself, so we don't have to ask user figure that out.